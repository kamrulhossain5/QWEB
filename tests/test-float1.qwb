function int main(){
  float a;
  a = 5.123456789;
  printf(a);
}
