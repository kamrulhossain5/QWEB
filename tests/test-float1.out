5.12346<br>
